File: Herencia/ejercicio 3/Python/Estudiante.py
from Persona import Persona
class Estudiante(Persona):
    def __init__(self, ci, nombre, apellido, celular, nacimiento, sexo, ru, ingreso, semestre):
        super().__init__(ci, nombre, apellido, celular, nacimiento, sexo)
        self.ru = ru
        self.ingreso = ingreso
        self.semestre = semestre
    def mostrar(self):
        super().mostrar()
        print(f"RU: {self.ru}, Semestre: {self.semestre}")

File: Herencia/ejercicio 3/Java/Estudiante.java
class Estudiante extends Persona {
    public String ru, fechaIngreso;
    public int semestre;
    public Estudiante(String ci, String nombre, String apellido, String celular, LocalDate fechaNac, String sexo,
                      String ru, String fechaIngreso, int semestre) {
        super(ci, nombre, apellido, celular, fechaNac, sexo);
        this.ru = ru;
        this.fechaIngreso = fechaIngreso;
        this.semestre = semestre;
    }
    @Override
    public void mostrar() {
        super.mostrar();
        System.out.println("RU: " + ru + ", Semestre: " + semestre);
    }
}